src/openrct2/scripting/ScriptEngine.cpp

```diff
@@ -1090,7 +1090,7 @@ GameActions::Result ScriptEngine::QueryOrExecuteCustomGameAction(const CustomAct
 
     auto action = GameActions::Result();
     action.Error = GameActions::Status::Unknown;
-    action.ErrorTitle = "Unknown custom action";
+    action.ErrorTitle = " [Unknown custom action] \n" + customAction.GetPluginName() + ": " + actionz;
```

Comment:
How about using ErrorMessage instead for the text after the new line?

Reply:
Yeah, that makes more sense. Thanks 🙂